config/sync/easy_breadcrumb.settings.yml

```diff
@@ -13,9 +13,3 @@ hide_single_home_item: false
 title_from_page_when_available: true
```

Comment:
Because you uninstalled easy_breadcrumb you will need to delete this file. Leaving it in will result in config import problems (you can't import config for a module that isn't installed)